src/platforms/binary64.rkt

```diff
@@ -105,8 +106,48 @@
                                    tgamma
                                    trunc)]
                             [(binary64 binary64 binary64)
-                             (atan2 copysign fdim fmax fmin fmod hypot pow remainder)]
-                            [(binary64 binary64 binary64 binary64) (fma)])
+                             (atan2 copysign fdim fmax fmin fmod pow remainder)])
+
+(define-libm c_erfc (erfc double double))
+(define-libm c_expm1 (expm1 double double))
+(define-libm c_log1p (log1p double double))
+(define-libm c_hypot (hypot double double double))
+(define-libm c_fma (fma double double double double))
```

Comment:
This is all fine but I wonder if it'd be useful to have a `#:optional` flag that just skips the `define-operator-impl` block if the `#:fl` is `#f`.